emscripten.py

```diff
@@ -2000,7 +2024,10 @@ def create_sending_wasm(invoke_funcs, forwarded_json, metadata, settings):
   implemented_functions = set(metadata['implementedFunctions'])
   global_funcs = list(set([key for key, value in forwarded_json['Functions']['libraryFunctions'].items() if value != 2]).difference(set(global_vars)).difference(implemented_functions))
 
-  send_items = basic_funcs + invoke_funcs + global_funcs + basic_vars + global_vars
+  jscall_funcs = ['jsCall_' + sig for sig in jscall_sigs]
+
+  send_items = basic_funcs + invoke_funcs + jscall_funcs + global_funcs + \
```

Comment:
Nit: Personally I prefer parentheses instead of backslashes for multi-line expressions. Not sure if Emscripten has a preferred style preference here. Also not sure if it's worth caring about.

Reply:
I'm not sure, this is pretty rare, but I think we have more backslashes. I'm ok either way.

Reply:
Changed to parentheses.